common/locales/en/generic.json

```diff
@@ -184,8 +184,8 @@
     "wonChallengeShare": "I won a challenge in Habitica!",
     "achievementShare": "I earned a new achievement in Habitica!",
     "orderBy": "Order By <%= item %>",
-
-    "you": "you",
```

Comment:
Is this used anywhere else? Or did _you_ add it for the original notifications pr?